.circleci/config.yml

```diff
@@ -225,6 +226,52 @@ jobs:
           paths:
             - .
 
+  build-macos-runtime:
+    macos:
+      xcode: "10.3.0"
+    environment:
+      - HERMES_WS_DIR: /tmp/hermes
+      - TERM: dumb
+      # Homebrew currently breaks while updating:
+      # https://discuss.circleci.com/t/brew-install-fails-while-updating/32992
+      - HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            brew install cmake ninja
+      - run:
+          name: Set up workspace
+          command: |
+            mkdir -p "$HERMES_WS_DIR" "$HERMES_WS_DIR/output" "$HERMES_WS_DIR/package-root"
+            ln -sf "$PWD" "$HERMES_WS_DIR/hermes"
+      - run:
+          name: Build Hermes for macOS as CocoaPods pod
```

Comment:
So the process for building a pod doesn't involve running the pod tool?

Reply:
A pod is either source or binaries with a podspec that knows how to include those. Typically the podspecs are served from a centralized spec repo, in which case you might use the `pod` tool to push the spec on release. In the case of RN, though, they are shipped within npm packages, so the only work that needs to be done is to build the binaries and package them.

Reply:
Since the podspec seems to have code for building the Framework (although just by calling cmake), is there a reason to write out the similar logic here?  It just seems like a violation of DRY.

Reply:
There is no `pod` cli feature to invoke an arbitrary podspec’s  `prepare_command`. I did try to counter the DRY violation by refactoring the podspec such that we could load it here and get the configuration that matters (I don’t consider the build and destroot locations to be part of that), so those only need to be maintained in a single place.

Reply:
Bummer.  I guess we could the logic in prepare_command into a shell script which is run by cocoapods or circleci.  but that can be a followup, too.  I've asked enough questions already.  Thanks for educating me.